Assets/tets.cs

```diff
@@ -1,18 +0,0 @@
-using System.Collections;
```

Comment:
thank you for deleting old useless files, its good practise to keep workspaces clean